pkg/gitversion/gitversion.go

```diff
@@ -268,22 +270,36 @@ func mostRecentTag(repo *git.Repository, ref plumbing.Hash) (bool, *plumbing.Ref
 // workTreeIsDirty returns whether the worktree associated with the given repository
 // has local modifications.
 func workTreeIsDirty(repo *git.Repository) (bool, error) {
-
 	debug := viper.GetBool("debug")
-
-	worktree, err := repo.Worktree()
+	workTree, err := repo.Worktree()
 	if err != nil {
-		return false, fmt.Errorf("error getting git worktree: %w", err)
+		return false, err
 	}
 
-	status, err := worktree.Status()
-	if err != nil {
-		return false, fmt.Errorf("error getting git worktree status: %w", err)
+	if _, ok := repo.Storer.(*filesystem.Storage); !ok {
```

Comment:
Technically this would only be possible in tests now. Perhaps should fall back to this if we can't find the git executable or something? Also would consider completely nuking this option for now (until go-git provides an alternative to do dirty detection)